inc/functions.php

```diff
@@ -4337,12 +4336,10 @@ function build_breadcrumb()
 				eval("\$nav .= \"".$templates->get("nav_bit")."\";");
 			}
 		}
+		$navsize = count($navbits);
+		$navbit = $navbits[$navsize-1];
 	}
 
-	$activesep = '';
```

Comment:
This should be retained as otherwise an undefined notice might be prompted due to `$activesep` being defined within a condition later on.

Reply:
Isn't it a duplicate assignment as this has already been declared here?:
https://github.com/mybb/mybb/blob/feature/inc/functions.php#L4297

Reply:
Wooops, good spot. I used GitHub's diff and didn't see the whole function, my bad 😄